packages/nimble-components/src/table-column/date-text/index.ts

```diff
@@ -82,7 +83,9 @@ export class TableColumnDateText extends TableColumnTextBase {
     @attr({ attribute: 'custom-format-matcher' })
     public customFormatMatcher: FormatMatcherAlgorithm;
 
-    @attr({ attribute: 'custom-hour12', mode: 'boolean' })
+    // Later versions of FAST (than the legacy branch we're on) have a nullableBooleanConverter.
```

Comment:
Could you link this code comment to #572 and add a note in that issue mentioning this code? That might help increase the odds of us fixing this when we upgrade.